domains/digit.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "nt-ossas",
+        "email": "ossas050288@gmail.com"
+    },
+    "record": {
+        "CNAME": "nt-ossas.github.io"
```

Comment:
404 page (if you have your site hosted then you can comment below a preview of the website)

Reply:
try https://nt-ossas.github.io/DIGIT/techzone/home
or https://nt-ossas.github.io/DIGIT/cryptozone/home

Reply:
The link you provided works, but you ned to edit that link in your CNAME for it to be considered reachable and complete.